src/diffusers/image_processor.py

```diff
@@ -452,7 +476,8 @@ def rgblike_to_depthmap(image: Union[np.ndarray, torch.Tensor]) -> Union[np.ndar
         """
         return image[:, :, 1] * 2**8 + image[:, :, 2]
 
-    def numpy_to_depth(self, images: np.ndarray) -> List[PIL.Image.Image]:
+    def numpy_to_pil_depth(self, images: np.ndarray) -> List[PIL.Image.Image]:
```

Comment:
Can we try to leave the name as is - otherwise this would break backwards compatibility

Reply:
sounds good